docs/writing-tests/server-features.md

```diff
@@ -126,4 +129,26 @@ API are documented in [Writing H2 Tests](h2tests).
 
 > <b>Important:</b> The HTTP/2.0 server requires you to have Python 2.7.10+
 and OpenSSL 1.0.2+. This is because HTTP/2.0 is negotiated using the
-[TLS ALPN](https://tools.ietf.org/html/rfc7301) extension, which is only supported in [OpenSSL 1.0.2](https://www.openssl.org/news/openssl-1.0.2-notes.html) and up.
+[TLS ALPN](https://tools.ietf.org/html/rfc7301) extension, which is only
+supported in
+[OpenSSL 1.0.2](https://www.openssl.org/news/openssl-1.0.2-notes.html) and up.
+
+
+### Tests Requiring QUIC
+
+We do not support loading a test over QUIC yet, but a test can establish a QUIC
+connection to the test server (e.g. for WebTransport, similar to WebSocket).
+Since the QUIC server is not yet enabled by default, tests need to explicitly
+declare the requirement:
```

Comment:
This sounds like if you declare the requirement then the QUIC server will be enabled, which is not true. I think it would help to explain *less* actually:

`Tests must declare that they need access to the QUIC server:`

And then below:

`The QUIC server is not yet enabled by default, so QUIC tests will be skipped unless --do-quic-things is specified to ./wpt run.```